itou/www/employee_record_views/views.py

```diff
@@ -238,8 +238,8 @@ def list_employee_records(request, template_name="employee_record/list.html"):
             .filter(status=status)
             .order_by(*employee_record_order_by)
         )
-        if job_seekers := filters_form.cleaned_data.get("job_seekers"):
-            data = data.filter(job_application__job_seeker__in=job_seekers)
+        if job_seeker := filters_form.cleaned_data.get("job_seeker"):
```

Comment:
```suggestion
        if job_seeker_id := filters_form.cleaned_data.get("job_seeker"):
```
ici aussi du coup :)

Reply:
Erf, oui. Merci !